src/uv-common.c

```diff
@@ -113,6 +113,8 @@ const char* uv_err_name(int err) {
   switch (err) {
     UV_ERRNO_MAP(UV_ERR_NAME_GEN)
     default:
+      /* @20170517-sanggyu: To figure out what error number is missing */
+      printf("uv_err_name for (%d) is missing. Add err name string\n", err);
```

Comment:
My two cents:
1) error messages should use the error channel (e.g. `fprintf(stderr, ...)`)
2) TDLOG should be used for such things
3) Why do we need to make this change permanent? This will increase the binary size (a little), but I don't see why we need it in release builds. Is this happening many times?